### ISSUE TITLE
Incorrect travel time if station depth is nonzero

### ISSUE THREAD

**Issue by seisman** [crotwell/TauP#2]
TauP version:
```
taup_time --version                    
edu.sc.seis:TauP:2.4.2 Fri Mar 10 17:38:18 EST 2017
```
This issue can be reproduces with following steps:

1. PcP travel time with receiver depth=0 km:
```
taup_time -mod prem -ph PcP -h 500 -deg 0   

Model: prem
Distance   Depth   Phase   Travel    Ray Param  Takeoff  Incident  Purist    Purist
  (deg)     (km)   Name    Time (s)  p (s/deg)   (deg)    (deg)   Distance   Name 
-----------------------------------------------------------------------------------
    0.00   500.0   PcP      450.81     0.000      0.00     0.00     0.00   = PcP
```

2. PcP travel time with receiver depth=200 km:
```
 taup_time -mod prem -ph PcP -h 500 -deg 0 --stadepth 200

Model: prem  Receiver Depth: 200.0 km
Distance   Depth   Phase   Travel    Ray Param  Takeoff  Incident  Purist    Purist
  (deg)     (km)   Name    Time (s)  p (s/deg)   (deg)    (deg)   Distance   Name 
-----------------------------------------------------------------------------------
    0.00   500.0   PcP      446.84     0.000      0.00     0.00     0.00   = PcP 
```

3. Travel time of upgoing P wave, with the source at 200 km and receiver at 0 km:
```
taup_time -mod prem -ph p -h 200 -deg 0                  

Model: prem
Distance   Depth   Phase   Travel    Ray Param  Takeoff  Incident  Purist    Purist
  (deg)     (km)   Name    Time (s)  p (s/deg)   (deg)    (deg)   Distance   Name 
-----------------------------------------------------------------------------------
    0.00   200.0   p         25.76     0.000    180.00     0.00     0.00   = p    
```

The result shows: `450.81 != 446.84 + 25.76`.

**Comment by crotwell** [comment]
Looks like a bug, most likely with the PcP to the station at depth.
Thanks for reporting this.

**Comment by crotwell** [comment]
Fixed in commit dd4e564d88a64fa1d7afb6061b8b8fdfb349583b
